docs/src/documentation/03-components/07-interaction/stepper/03-accessibility.mdx

```diff
@@ -52,4 +61,33 @@ The screen reader will announce the value title (`Number of passengers`) and but
 </Stack>
 ```
 
-This example includes `ariaLabelledby` prop. In this case, `ariaLabelledBy` prop is prioritized over `ariaLabelValue`, so the screen reader will announce the value title (`Passengers`) and buttons title (`Add a passenger`, `Remove a passenger`) once they are focused by the screen reader.
+This example includes `ariaLabelledby` prop. In this case, `ariaLabelledby` prop is prioritized over `ariaLabelValue`, so the screen reader will announce the value title (`Passengers`) and buttons title (`Add a passenger`, `Remove a passenger`) once they are focused by the screen reader.
+
+```jsx
+<Stack>
+  <Stack>
+    <Text id="adults-title">Adults</Text>
+    <Text id="adults-description">Number of passengers in your group</Text>
+  </Stack>
+  <Stepper
+    step={1}
+    minValue={0}
+    minValue={10}
+    ariaLabelValue="Number of passengers"
+    ariaLabelledby="adults-title"
+    ariaDescribedby="adults-description"
+    titleDecrement="Remove a passenger"
+    titleIncrement="Add a passenger"
+    descriptionIncrement="adults-title"
+    descriptionDecrement="adults-title"
+  />
+</Stack>
+```
+
+This example includes `ariaLabelledby`, `descriptionIncrement` and `descriptionDecrement` props.
+
+When decrement button is focused by screen reader, the screen reader will announce the button title (`Remove a passenger`) and complementary information (`Adults`).
+
+For input field, the value of `ariaLabelValue` (`Number of passengers`) is read first, then the complementary information (value of `ariaDescribedby` prop) is announced - `Number of adults in your group`. Value of `ariaLabelledby` is ignored in this case.
```

Comment:
Why is value of `ariaLabelledby` ignored if it's the same case as in the previous example?